src/components/index.js

```diff
@@ -1 +1,2 @@
-export { default as Button } from './Button';
\ No newline at end of file
+export { default as Button } from './Button';
+export { default as Input } from './Input';
```

Comment:
Sempre deixe uma linha no final
```suggestion
export { default as Input } from './Input';

```

Reply:
Não entendi.

Reply:
Sempre deixe uma linha em branco no final de todo o arquivo

Reply:
fiz isso em todos os arquivos, abri de um por um e verifiquei